### ISSUE TITLE
Publish DMCA Request

### ISSUE THREAD

**Issue by jcalfee** [Chocobozzz/PeerTube#1406]
Just a tip: Make the DCMA request public so everyone can review.  This opens the door for the legality of the DCMA request to be reviewed.  Also gives people a chance to reject the request with evidence, move the material, or just allow it to fall off the radar and get deleted.

Some material subject to the DCMA request may be evidence in a victim-based crime involving harm or violence.  So the DCMA request could be part of an attempt to cover-up corporate or government level crimes.  This happens frequently especially among users that use media to expose wrong-doing.  To make it more interesting, DCMA requests are of course mixed in with things that generally involve mundane things that people are not concerned with (spam if you will).

Obviously complex in the final decision, so a simple solution is to allow the DCMA requests to be publicly reviewed for N days before going into affect.  I suggest a DCMA request feed giving the server admin the ability to prompt for and publish or not publish arbitrary name value pair fields in an API, I think you'll have a very robust platform.  

A template of fields will make that easy..

- Publish DCMA request (y/n - either way, tell the DCMA submitter if it is public or private)
- Add field: name, value (y/n - publish) - each field, admin says: public, private, reserves right to publish
- Days for admin to review prior to auto-publish
- Days for community review
- Allow users to comment on the DCMA request
- If material is removed, keep DCMA request (y/n)

If you think a question is sensitive, do not provide a default..

**Comment by jcalfee** [comment]
Yes, Digital Millennium Copyright Act .. A plug-in sounds better.  I don't want to see it implied though (lol).  I think users should decide if they should or should not see something (not the law or regulators).  If you obey to cover up a crime it is not a good thing, that is a bad thing..

I just wanted to though out the idea: make any request like that public.

**Comment by robinkooli** [comment]
@Kazer67 Companies from European Union and European Economic Area send you an European Intellectual Property Rights Enforcement Directive request.

**Comment by JigmeDatse** [comment]
This is something which is really important right now.  Currently we have been hit with a DMCA/Rights Violation on videos we do not host, which has very basic boilerplate information.  

We have posted a video on the instance about it, but feel that this is *not* the place to share it.  Feel free to directly contact us.  Our instance is video.datsemultimedia.com and the account is jigme.  This issue is not recent, but I believe that it's probably worth looking to see how to deal with copyright claims.

**Comment by jcalfee** [comment]
Also, keep in mind that the reporter of the DMCA may come back and want to remove their claim.

**Comment by Chocobozzz** [comment]
Merging this issue in https://github.com/Chocobozzz/PeerTube/issues/7198 (that is more recent but with an interesting discussion)
